### PR TITLE
changed home carousel

### PR DESCRIPTION
**Pull request by Puneet-Amrutkar**
made the home carousel same as that of the events page

### CODE REVIEW

File: src/Assets/Images/homebg.png
Comment (aryav-022) [inline src/Assets/Images/homebg.png:1]: ig this image is not used

**Review by aryav-022 — APPROVED** [review]
good to go

File: src/Assets/Images/homebg.png
Comment (Puneet-Amrutkar) [reply src/Assets/Images/homebg.png:1]: yes... deleting it...

Comment (Puneet-Amrutkar) [reply src/Assets/Images/homebg.png:1]: was just trying a new landing page bg

**Comment by Puneet-Amrutkar** [comment]
deleted the useless file

I guess changes good to merge

**Pull request merged**